frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -325,133 +404,183 @@ export const OperandList: React.FC<OperandListProps> = (props) => {
       data={data}
       EmptyMsg={EmptyMsg}
       aria-label="Operands"
-      Header={Header}
-      Row={OperandTableRow}
+      Header={managesOtherNamespaces && props.toggleView ? AllNsHeader : CurrentNsHeader}
+      Row={(listProps) => (
+        <OperandTableRow {...listProps} olmObj={olmObj} toggleView={toggleView} />
+      )}
       virtualize
     />
   );
 };
 
+const getFirehoseResources = (
```

Comment:
Remove reference to legacy firehose name. "getK8sWatchResources" or something along those lines.